server/docs/authors-haven-api.yml

```diff
@@ -825,3 +862,71 @@ components:
         userName:
           type: string
           example: prider
+    createArticleResponse:
```

Comment:
@Adekoreday I tested your PR and realized that you didn't provide a means to upload the image that will cater to the "image" attribute you have on the Article model in the documentation. Kindly please modify your PR accordingly.